waspc/src/Wasp/AppSpec/App/EmailSender.hs

```diff
@@ -15,11 +16,23 @@ data EmailSender = EmailSender
   }
   deriving (Show, Eq, Data)
 
-data EmailProvider = SMTP | SendGrid | Mailgun
+data EmailProvider = SMTP | SendGrid | Mailgun | Dummy
   deriving (Eq, Data, Show)
 
 data EmailFromField = EmailFromField
   { name :: Maybe String,
     email :: String
   }
   deriving (Show, Eq, Data)
+
+defaultDummyEmailSender :: EmailSender
```

Comment:
Usually we prefer putting default values into Generator, to keep AppSpec simple. Maybe that is the right move here also.

Reply:
Makes sense 👍